pkg/domain/infra/abi/play.go

```diff
@@ -36,7 +32,15 @@ import (
 	"github.com/opencontainers/selinux/go-selinux"
 	"github.com/sirupsen/logrus"
 	yamlv3 "gopkg.in/yaml.v3"
+	"io"
```

Comment:
I think you may want to ensure your IDE is running `goimports` along with `gofmt` on save.